Source/FFmpegMediaSource.cpp

```diff
@@ -1047,27 +1049,65 @@ namespace winrt::FFmpegInteropX::implementation
         }
         if (currentAudioStream)
```

Comment:
Since we now store effects per-stream, all non-specific methods should set/clear filters in all the appropriate streams, not just the current one. The get method should just return the filter from the current stream (we can't do better than that). The variable currentAudioEffects is not needed anymore and can be removed.

Setting filters in a disabled stream is not a resource or performance issue, since the effect graph is only created when a sample for that stream is requested.

Reply:
I missed that. Fixed.